### ISSUE TITLE
--include doesn't support external package(s) ?

### ISSUE THREAD

**Issue by iurii-ssv** [ferranbt/fastssz#179]
Perhaps I'm using it wrong (there isn't a proper example for it though), but when doing something like
```
import (
    spectypes "github.com/ssvlabs/ssv-spec/types"
)

//go:generate sszgen -path ./shares.go --objs storageShare --include github.com/ssvlabs/ssv-spec/types

type storageShare struct {
	OperatorID spectypes.OperatorID // uint64 essentially
}
```
I get the following error running `sszgen`:
```
[ERR]: stat github.com/ssvlabs/ssv-spec/types: no such file or directory
shares.go:18: running "sszgen": exit status 1
```
I've tried different ways to reference this external package (using alias and whatnot) but result is the same.

**Comment by ferranbt** [comment]
For `include` to work you have to use the absolute path for `github.com/ssvlabs/ssv-spec/types`. The code generator does not resolve paths itself.

**Comment by iurii-ssv** [comment]
> For include to work you have to use the absolute path for github.com/ssvlabs/ssv-spec/types. The code generator does not resolve paths itself.

Hmm, well, but that doesn't really make much sense in terms of portability - local path on my machine would be different from the one my colleague uses (and I think it also might change for new package version). So it pretty much means `generator` can't be used with external packages, right ?

**Comment by mcdee** [comment]
Bear in mind that includes can be relative, so if you use the standard go location for packages your include can be something like `../../repo/package/...` and it will work.

**Comment by iurii-ssv** [comment]
> Bear in mind that includes can be relative, so if you use the standard go location for packages your include can be something like `../../repo/package/...` and it will work.

Right, but that's still probably "a hacky way to do it" at best - because, like I mentioned above:
- different version of the same package might have different "local" file path (meaning once I upgrade to a newer version of some package `generator` was using external type from - `generator` will no longer find it, and I'd need to manually resolve it again)
- people might not "use defaults" when it comes to where golang (and it's tooling) was/is installed
- ...

To me it seems the best work-around at the moment would be to redefine those external types in the package `ssz generator` runs in (and manually type-convert those)
